src/files.js

```diff
@@ -245,6 +245,26 @@ export function getFilePath (cozy, file = {}, folder) {
   return `${folderPath}${file.name}`
 }
 
+export function getShareLink (cozy, id) {
+  if (!id) {
+    return Promise.reject(Error('An id should be provided to create a share link'))
```

Comment:
Why avoiding the `new` keyword ? Is something we should generalize in all cozy-client-js ?